MovieApp/LogInViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  LogInViewController.swift
+//  MovieApp
+//
+//  Created by endava-bootcamp on 28.03.2023..
+//
+
+import Foundation
+import UIKit
+import PureLayout
+import MovieAppData
+
+class LogInViewController: UIViewController {
+    
+    private var mainLabel: UILabel!
+    private var emailLabel: UILabel!
+    private var passLabel: UILabel!
+    private var emailField: UITextField!
```

Comment:
Convention in naming - `emailTextField` so the name of the property is self explanatory. `EmailField` doesn't say much and it's not clear.